src/AuthKeyValueUser.php

```diff
@@ -247,12 +239,19 @@ public function getPropertyValues($property)
                     $returnValue = array($value);
                 }
             }
+
+            if (!empty($returnValue)) {
+                $returnValue = json_decode(current($returnValue), true);
+
+                if (!is_array($returnValue)) {
+                    $returnValue = [$returnValue];
+                }
```

Comment:
From what I see - it's always an array, so this `if` block is not needed :upside_down_face:

Reply:
It in fact depends

Considering `member` (isMultiple() === true) property, the result of decode is an array

```
$user->getPropertyValues('http://www.tao.lu/Ontologies/TAOGroup.rdf#member')

/var/www/html/vendor/oat-sa/generis-auth-keyvalue/src/AuthKeyValueUser.php:246:
array (size=2)
  0 => string 'https://test-nfer.docker.localhost/ontologies/tao.rdf#i60afb7d214d439441f02b086f89837' (length=85)
  1 => string 'https://test-nfer.docker.localhost/ontologies/tao.rdf#i60afb657ce80234247f49b80963a15e' (length=86)
```

But for `UpdatedAt` (isMultiple() === false), the result of decode is a string

```
$user->getPropertyValues('http://www.tao.lu/Ontologies/TAO.rdf#UpdatedAt')

/var/www/html/vendor/oat-sa/generis-auth-keyvalue/src/AuthKeyValueUser.php:246:string '1622657899.7188' (length=15)
```

That`s why I would prefer to move groups from `auth:user:extra_parameters` to `auth:user` key, to not have to mess in this logic :)